src/metabase/task/send_pulses.clj

```diff
@@ -101,6 +101,34 @@
   (let [{:strs [pulse-id channel-ids]} (qc/from-job-data context)]
     (clear-pcs-and-send-pulse! pulse-id channel-ids)))
 
+;;; ------------------------------------------------ Job: InitSendPulseTriggers ----------------------------------------------------
+
+(declare update-send-pulse-trigger-if-needed!)
+
+(defn- init-send-pulse-triggers!
+  []
+  (let [trigger-slot->pc-ids (as-> (t2/select :model/PulseChannel :enabled true) results
+                               (group-by #(select-keys % [:pulse_id :schedule_type :schedule_day :schedule_hour :schedule_frame]) results)
```

Comment:
Can we group simply by `pulse_id` instead? I'm concerned that we'll do a migration on the `:schedule_` columns in the future and leaving them here will be confusing
```suggestion
                               (group-by :pulse_id results)
```

Reply:
I don't this removes the confusion from the modeling perspective. once we migrate to corn string we can group by it by then.